x/stakeibc/keeper/icqcallbacks_callibrate_delegation.go

```diff
@@ -97,6 +70,12 @@ func CalibrateDelegationCallback(k Keeper, ctx sdk.Context, args []byte, query i
 	}
 
 	delegationChange := validator.Delegation.Sub(delegatedTokens)
+	// if the delegation change is more than the calibration threshold constant, log and throw an error
+	if delegationChange.Abs().GTE(CalibrationThreshold) {
+		k.Logger(ctx).Error(utils.LogICQCallbackWithHostZone(chainId, ICQCallbackID_Calibrate,
+			"Delegation change is GT CalibrationThreshold, failing calibration callback"))
+		return nil
```

Comment:
Just double checking - no state is written above this line right? Doesn't look like it to me, but since this doesn't throw an error, state above this line could be written

Reply:
Good flag. Correct, no state is written in the callback above this line.